patches/chrome-installer-mini_installer-mini_string.cc.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/chrome/installer/mini_installer/mini_string.cc b/chrome/installer/mini_installer/mini_string.cc
+index 6bdfd96258abdf211b9993f27bff5cecd31d175f..4ee7fdbb15db4dc73c741ce2bcdc0dfb4a94da8f 100644
+--- a/chrome/installer/mini_installer/mini_string.cc
++++ b/chrome/installer/mini_installer/mini_string.cc
+@@ -21,6 +21,20 @@ bool EqualASCIICharI(wchar_t a, wchar_t b) {
+ 
+ namespace mini_installer {
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
++// Coverts a string in place to uppercase
++void SafeStrASCIIUpper(wchar_t* str, size_t size) {
```

Comment:
If this function is only used by `mini_installer.cc`, I think defining there seems better to avoid adding new patch file.